sql/src/test/java/io/crate/integrationtests/OptimizeTableIntegrationTest.java

```diff
@@ -223,4 +218,36 @@ public void testOptimizeMultipleTablesWithPartition() {
         execute("select * from t1");
         assertThat(response.rowCount(), is(2L));
     }
+
+    @Test
+    public void testOptimizeTableWithInvalidParamName() {
```

Comment:
i moved these 4 tests from analyzer to integration, instead of going with the `<Plan>.createStatement` pattern as it is done in alter and create table analyzer test setups. thought that it would be cleaner.

Reply:
I think we should find a way to extend our unit-test framework so that we can test the parts we're interested in without having to setup nodes + networking and real tables with shards etc..

We're talking about at least 1 to 2 orders of magnitude of difference in execution speed here.

Reply:
I agree that the `<Plan>.createStatement` pattern is not perfect, on the other side I'd like to not convert unit test into integration tests when not really needed. If if simple integration tests increases overall test runtime quite more than unit test.
@mfussenegger any opinion on that?

Reply:
Updated the pr to follow the `<Plan>.createStatement` pattern.